tools/osemosys/features.yaml

```diff
@@ -0,0 +1,401 @@
+        - https://doi.org/10.1016/j.isci.2025.111792
+      value: dev
+  system__carriers:
+    coupled:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+      value: y
+    decoupled:
+      source: []
+      value: n
+  system__limits:
+    capacity:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#new-capacity-constraints
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#total-capacity-constraints
+      value: y
+    operation:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#annual-activity-constraints
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#total-activity-constraints
+      value: y
+  time__foresight:
+    myopic:
+      source:
+        - https://github.com/KTH-dESA/OSeMOSYS_step
+        - https://doi.org/10.1186/s13705-024-00469-w
+      value: dev
+    perfect:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#objective-function
+      value: y
+  time__horizon:
+    multi_year:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#global-parameters
+      value: y
+    single_year:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#global-parameters
+      value: y
+    sub_day:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#global-parameters
+      value: y
+    sub_year:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#global-parameters
+      value: y
+  time__resolution__simplification:
+    distortion_mitigation:
+      source: []
+      value: n
+    load_duration_curves:
+      source: []
+      value: n
+    resampling:
+      source: []
+      value: n
+    typical_periods:
+      source:
+        - https://doi.org/10.1016/j.enpol.2011.06.033
+      value: y
+  transmission__limits:
+    gtc:
+      source: []
+      value: y
+    phase_angle:
+      source: []
+      value: n
+    ptdf:
+      source: []
+      value: n
+    thermal:
+      source: []
+      value: n
+  transmission__opf:
+    ac_opf:
+      source: []
+      value: n
+    dc_opf:
+      source: []
+      value: n
+    sec_opf:
+      source: []
+      value: n
+    transport:
+      source: []
+      value: y
```

Comment:
I assume this is true for OSeMOSYS, i.e. any transmission/distribution is a simple "energy transport" problem

Reply:
Yep. Added source.